src/main/java/world/bentobox/caveblock/Settings.java

```diff
@@ -26,1898 +32,1894 @@
+     * @return true if water is not safe in this world, e.g, should not be a home location
+     */
+    @Override
+    public boolean isWaterUnsafe()
+    {
+        return false;
+    }
+
+
+    /**
+     * This method returns the resetEpoch object.
+     * @return the resetEpoch object.
+     */
+    @Override
+    public long getResetEpoch()
+    {
+        return resetEpoch;
+    }
+
+
+    /**
+     * This method returns the worldDepth value.
+     * @return the value of worldDepth.
+     */
+    public int getWorldDepth()
+    {
+        return worldDepth;
+    }
+
+
+    /**
+     * This method returns the normalRoof value.
+     * @return the value of normalRoof.
+     */
+    public boolean isNormalRoof()
+    {
+        return normalRoof;
+    }
+
+
+    /**
+     * This method returns the normalFloor value.
+     * @return the value of normalFloor.
+     */
+    public boolean isNormalFloor()
+    {
+        return normalFloor;
+    }
+
+
+    /**
+     * This method returns the normalMainBlock value.
+     * @return the value of normalMainBlock.
+     */
+    public Material getNormalMainBlock()
+    {
+        return normalMainBlock;
+    }
+
+
+    /**
+     * This method returns the normalBlocks value.
+     * @return the value of normalBlocks.
+     */
+    public List<String> getNormalBlocks()
+    {
+        return normalBlocks;
+    }
+
+
+    /**
+     * This method returns the netherFloor value.
+     * @return the value of netherFloor.
+     */
+    public boolean isNetherFloor()
+    {
+        return netherFloor;
+    }
+
+
+    /**
+     * This method returns the netherMainBlock value.
+     * @return the value of netherMainBlock.
+     */
+    public Material getNetherMainBlock()
+    {
+        return netherMainBlock;
+    }
+
+
+    /**
+     * This method returns the netherBlocks value.
+     * @return the value of netherBlocks.
+     */
+    public List<String> getNetherBlocks()
+    {
+        return netherBlocks;
+    }
+
+
+    /**
+     * This method returns the endRoof value.
+     * @return the value of endRoof.
+     */
+    public boolean isEndRoof()
+    {
+        return endRoof;
+    }
+
+
+    /**
+     * This method returns the endFloor value.
+     * @return the value of endFloor.
+     */
+    public boolean isEndFloor()
+    {
+        return endFloor;
+    }
+
+
+    /**
+     * This method returns the endMainBlock value.
+     * @return the value of endMainBlock.
+     */
+    public Material getEndMainBlock()
+    {
+        return endMainBlock;
+    }
+
+
+    /**
+     * This method returns the endBlocks value.
+     * @return the value of endBlocks.
+     */
+    public List<String> getEndBlocks()
+    {
+        return endBlocks;
+    }
+
+
+    /**
+     * This method returns the numberOfBlockGenerationTries value.
+     * @return the value of numberOfBlockGenerationTries.
+     */
+    public int getNumberOfBlockGenerationTries()
+    {
+        return numberOfBlockGenerationTries;
+   
```

Comment:
This is just so you can find out where the entities are spawning so you can check them out by teleporting there.